app/webpacker/styles/sections/hero.scss

```diff
@@ -1,6 +1,18 @@
 // layout, mobile-first
 $mobile-cutoff: 800px;
 
+@mixin safari-support {
```

Comment:
I didn't know you could do this in Sass, it's really nice!

Would `safari-only` be a more-descriptive name?